src/xmlutil.rs

```diff
@@ -250,4 +282,23 @@ mod tests {
         }
     }
 
+    #[test]
+    fn skip_tree_with_subtree() {
+        let file = File::open("tests/sample-data/skip_tree.xml").unwrap();
+        let file = BufReader::new(file);
+        let mut my_parser  = EventReader::new(file);
+        let my_stack = my_parser.events().peekable();
+        let mut reader = XmlResponseFromFile::new(my_stack);
+
+        // skip two leading fields since we ignore them (xml declaration, return type declaration)
+        reader.next();
+        reader.next();
```

Comment:
I was wrong. The problem was with the timestamp type.

In JSON (used by DynamoDB), timestamps are numbers. EC2 returns timestamps in ISO 8601 format.

In d2ae714 I added a new method in the `GeneratorProtocol` trait. Thus, every generator can say what type has to be used for dates.